lib/mongoid/shell/commands/base.rb

```diff
@@ -28,7 +28,7 @@ def vargs(args = {})
             value = send(property)
             next unless value
             case value
-            when Boolean, TrueClass then key
+            when TrueClass then key
```

Comment:
I was puzzled by this check on a `Mongoid::Boolean` type here - not sure why you'd pass a Mongoid Boolean field type in here and why it would automatically evaluate to true.